cloud/cloud.cfg.d/99_wsl.cfg

```diff
@@ -1,4 +1,8 @@
 datasource_list: [WSL, None]
 network:
   config: disabled
-
+# Prevents creation of an unexpected user
+# See: https://github.com/ubuntu/WSL/issues/468
+system_info:
+  distro: ubuntu
```

Comment:
Is the distro key required? If not, I suggest avoiding it as we are in the "ubuntu" context already with that package.

Reply:
We need to match the exact keys in `/etc/cloud/cloud.cfg` otherwise this would be a generic `system_info` while the one in there would be the Ubuntu-specific one.
![image](https://github.com/ubuntu/wsl-setup/assets/11138291/a5f2aa7a-c702-4017-9f1a-eef68850259f)

Reply:
Makes sense! Mind adding this in the comment so that we don’t remove it by mistake later?